iac/main/resources/global.yml

```diff
@@ -40,6 +40,36 @@ GlobalLogBucketPolicy:
           Resource: !Sub ${GlobalLogBucket.Arn}/*
           Principal:
             AWS: !Sub arn:aws:iam::${AWS::AccountId}:root
+        - Effect: Allow
+          Action: s3:PutObject
+          Principal:
+            Service: logging.s3.amazonaws.com
+          Resource: !Sub ${GlobalLogBucket.Arn}/*
+          Condition:
+            ArnLike:
+              aws:SourceArn:
+                - !GetAtt AthenaWorkgroupBucket.Arn
```

Comment:
do all the buckets start with a common prefix, if so we can probably make this list a bit smaller

Reply:
yes that's a fair point i think they all start with `${Environment}-dap-` so i could use that. will confirm and implement if so thanks for the idea